### PR TITLE
Add ogre mythical creature

### PR DESCRIPTION
**Pull request by Ecksi**
One more mythical creature for everyone to enjoy

### CODE REVIEW

File: mythical-creatures/test/ogre-test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+var assert = require('chai').assert;
+var { Ogre, Human } = require('../exercises/ogre');
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: Change to `{Ogre, Human}` to match convention of existing similar mythical creatures

File: mythical-creatures/test/ogre-test.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    var ogre = new Ogre();
+    assert.isObject(ogre);
+  });
+
+  it.skip('should have a name', function () {
+    var ogre = new Ogre('Brak');
+    assert.equal(ogre.name, 'Brak');
+  });
+
+  it.skip('should live somewhere by default', function () {
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: Where should it live by default?

File: mythical-creatures/test/ogre-test.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  });
+
+  it.skip('it can meet humans', function() {
+    var ogre = new Ogre('Brak');
+    var human = new Human('Jane');
+
+    assert.equal(human.name, 'Jane');
+    
+    ogre.encounter(human);
+    assert.equal(human.encounter_counter, 1);
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: Change to camelCase since this is js.

File: mythical-creatures/test/ogre-test.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  });
+
+  it.skip('humans should only notice an ogre every third encounter', function () {
+    var ogre = new Ogre('Brak');
+    var human = new Human('Jane');
+
+    ogre.encounter(human);
+    ogre.encounter(human);
+    assert.equal(human.notices_ogre(), false);
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: same here and below.

File: mythical-creatures/test/ogre-test.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    assert.equal(human.notices_ogre(), true);
+  });
+
+  it.skip('can swing a club', function () {
+    var ogre = new Ogre('Brak');
+    var human = new Human('Jane');
+
+    ogre.swing_at(human);
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: and here.

File: mythical-creatures/test/ogre-test.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    var ogre = new Ogre('Brak');
+    var human = new Human('Jane');
+
+    for (i = 0; i <= 6; i++) {
+      ogre.encounter(human);
+    }
+
+    assert.equal(ogre.encounter_counter, 6);
+    assert.equal(ogre.swings, 2);
+    assert.equal(human.knocked_out, true);
```
Comment (damwhit) [inline mythical-creatures/test/ogre-test.js]: and here.

File: mythical-creatures/test/ogre-test.js (hunk 2 above)
Comment (Ecksi) [reply mythical-creatures/test/ogre-test.js]: It should live in a 'Swamp' by default.

**Comment by Ecksi** [comment]
Great, I'll make those changes this afternoon and re-PR. Thanks for taking a look!

**Comment by damwhit** [comment]
Sounds good. And of course! Nice work!

**Comment by damwhit** [comment]
Nice work! 🎉

**Pull request merged**